crates/task-impls/src/view_sync.rs

```diff
@@ -465,6 +465,13 @@ impl<
                     return;
                 }
 
+                // cancel poll for votes
+                self.network
+                    .inject_consensus_info(ConsensusIntentEvent::CancelPollForViewSyncVotes(
```

Comment:
I think it was correct before, we just timed out consensus for the round and are now maybe entering view sync, we should cancel polling for quorum votes and proposals but that should be handled in the consensus task